swap_meet/vendor.py

```diff
@@ -1,2 +1,78 @@
+from swap_meet.item import Item 
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+    def get_by_id(self, item_id): 
+        for item in self.inventory:
+            if item_id == item.id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
```

Comment:
Don't forget to add spaces between code blocks that have different responsibilities to help better visualize SRP 
```suggestion
        if my_item not in self.inventory or their_item not in other_vendor.inventory:
            return False
            
```

Reply:
Ok, we will!